sources/packages/backend/apps/api/Dockerfile.dev

```diff
@@ -1,5 +1,5 @@
 # Base Image
-FROM node:18.17.1-alpine3.18
+FROM node:22.13.1-alpine3.20
```

Comment:
Is the the [RedHat image](https://catalog.redhat.com/software/containers/ubi9/nodejs-22/66431d1785c5c3a31edd24f1?container-tabs=packages) based on `nodejs-22.11.0-1`?

If yes, we can have the `.dev` versions mirroring it as close as possible, if that makes sense.
Should we use the [22.11.0-alpine3.20](https://hub.docker.com/layers/library/node/22.11.0-alpine3.20/images/sha256-acfe48528d337d95603fbeaf249da67a07c4082e22f2c6ec85cf0fdffcfd63ff)?

Reply:
The point is not to use the most updated one but to mirror as much as possible the one that we have the apps deployed to and also execute the E2E tests in a closer version from the deployed ones.